xdsl/ir.py

```diff
@@ -1612,3 +1613,129 @@ def is_structurally_equivalent(
         ):
             return False
         return True
+
+
+class _AffineExprKind(Enum):
+    Add = ("+",)
```

Comment:
Is there a reason you are using tuple values, instead of just `"+"`?

Reply:
Python Enums don't allow that sadly. I replaced it with auto() and a get_token() function instead.